integration/src/main/java/scripts/client/CatalogClient.java

```diff
@@ -39,7 +34,8 @@ public CatalogClient(ServerSpecification server, TestUserSpecification testUser)
 
     if (testUser != null) {
       GoogleCredentials userCredential =
-          AuthenticationUtils.getDelegatedUserCredential(testUser, TEST_USER_SCOPES);
+          AuthenticationUtils.getDelegatedUserCredential(
+              testUser, AuthenticationUtils.userLoginScopes);
```

Comment:
Where are these userLoginScopes set from? (just learning question)

Reply:
`AuthenticationUtils` is from the terra testrunner library, and contains some code commonly needed when writing tests. `AuthenticationUtils.userLoginScopes` is a hardcoded list of OAuth scopes that's typically used when a user logs in. This API "logs in" `testUser` using these scopes and returns a credential, which is then used to get an access (aka Bearer) token. Once we have a bearer token, we can make authenticated API requests as `testUser`.

These scopes are the same as specified in our openapi yaml: https://github.com/DataBiosphere/terra-data-catalog/blob/main/common/openapi.yml#L281